rust-features.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
```suggestion
#!//usr/bin/env bash
```

There is no convention that `/bin/bash` exists.